### PR TITLE
chore: Removed front end from config

### PR DESCRIPTION
**Pull request by daviroo**
# Remove Frontend Service from Kubernetes Configuration

## Description
This PR removes the frontend service from our Kubernetes configuration. We've decided to host our frontend on Vercel instead of within the Kubernetes cluster.

## Changes
- Deleted `frontend.yaml` file
- Updated `README.md` to remove references to the frontend service
- Updated deployment instructions in `README.md`

## Rationale
Hosting our frontend on Vercel provides several benefits:
- Improved performance and global CDN
- Simplified deployment process
- Better separation of concerns between frontend and backend

## Testing
- Verified that all other services deploy correctly without the frontend service
- Confirmed that the README instructions are clear and accurate

## Next Steps
- Update any documentation or scripts that may have referenced the Kubernetes-hosted frontend

## Checklist
- [x] Updated README.md
- [x] Removed frontend.yaml
- [x] Tested deployment of remaining services

Please review and let me know if any further changes are needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This repository contains the Kubernetes configuration files for the hxckr projec
 ## Components
 
 1. **Server** (`server.yaml`): Main application server handling CRUD operations and business logic.
-2. **Frontend** (`frontend.yaml`): User interface for the application. Exposed externally. Communicates internally with the server.
+2. **Git Auth Service** (`git-auth-service.yaml`): Handles authentication for Git operations.
```
Comment (Extheoisah) [inline README.md]: Do we have a git-auth-service?

Comment (Extheoisah) [reply README.md]: Atleast I don't see a yaml file for it

Comment (daviroo) [reply README.md]: Good catch I was just testing to see if you would notice (really it's because I got confused by the git diff and left it in 😉 )

Comment (daviroo) [reply README.md]: Fixed in [chore: Removed unused git auth service from readme](https://github.com/Extheoisah/hxckr-k8s-bucket/pull/5/commits/9a8cb50a076ee91fd44a9dcdd1c94c79f356f8ff)

File: README.md [hunk 2]
```diff
@@ -31,7 +30,7 @@ Follow these steps to deploy the hxckr application:
 
 2. Apply the configurations:
    ```
-   kubectl apply -f server.yaml -f frontend.yaml -f softserve.yaml -f webhook-handler.yaml -f job-queue.yaml -f test-runners.yaml -f git-service.yaml -n hxckr
+   kubectl apply -f server.yaml -f git-auth-service.yaml -f softserve.yaml -f webhook-handler.yaml -f job-queue.yaml -f test-runners.yaml -f git-service.yaml -n hxckr
```
Comment (Extheoisah) [inline README.md]: Same here. No `git-auth-service.yaml`
```suggestion
   kubectl apply -f server.yaml -f softserve.yaml -f webhook-handler.yaml -f job-queue.yaml -f test-runners.yaml -f git-service.yaml -n hxckr
```

**Review by Extheoisah — APPROVED** [review]

**Pull request merged**